src/libstrongswan/plugins/botan/botan_util_keys.c

```diff
@@ -232,6 +265,18 @@ private_key_t *botan_private_key_load(key_type_t type, va_list args)
 	{
 		this = botan_ed_private_key_adopt(key);
 	}
+	else
+#endif
+#ifdef BOTAN_HAS_ML_DSA
+	if (streq(name, "ML-DSA") && (type == KEY_ANY || type == KEY_ML_DSA_44 ||
+							  type == KEY_ML_DSA_65 || type == KEY_ML_DSA_87))
```

Comment:
```suggestion
								  type == KEY_ML_DSA_65 || type == KEY_ML_DSA_87))
```

Reply:
Actually, this one is still the same, the public key one above was possibly fixed.